docs/migrations/version-0-to-1.md

```diff
@@ -36,5 +36,23 @@ These are all the preset hook changes:
 - C#, enum preset hooks had access to `model: CommonModel` which has now been changed to `model: ConstrainedEnumModel`
 
 General changes:
-- Hooks that gave access to properties/fields formally had the arguments `property`, `propertyName` and `type`, these are now wrapped within the [`ConstrainedObjectPropertyModel`](./internal-model.md#the-constrained-meta-model) and can be accessed through the `property` argument.
-- Hooks that gave access to enum items, now has the type [`ConstrainedEnumValueModel`](./internal-model.md#the-constrained-meta-model).
\ No newline at end of file
+- Hooks that gave access to properties/fields formally had the arguments `property`, `propertyName` and `type`, these are now wrapped within the [`ConstrainedObjectPropertyModel`](../internal-model.md#the-constrained-meta-model) and can be accessed through the `property` argument.
+- Hooks that gave access to enum items, now has the type [`ConstrainedEnumValueModel`](../internal-model.md#the-constrained-meta-model).
+
+## TypeScript and JavaScript module system
```

Comment:
OBS: Ended up having to move the module system option around for presets to access it 🙂 Which I had to ripple to the JavaScript generator as well.